docs/guide/src/extension.md

```diff
@@ -0,0 +1,47 @@
+# Using the web extension
+
+This section describes how to use the Penumbra Wallet web extension, a GUI client for Penumbra.
+
+## Installing the extension
+
+The Penumbra Wallet web extension only supports the Google Chrome browser.
+You must run Chrome in order to follow the instructions below.
+
+1. Visit the [Web Store page for the Penumbra Wallet](https://chrome.google.com/webstore/detail/penumbra-wallet/lkpmkhpnhknhmibgnmmhdhgdilepfghe),
+   and click **Add to Chrome** to install it.
+2. Navigate to the dApp website for the extension: [https://app.testnet.penumbra.zone/](https://app.testnet.penumbra.zone/) and click **Connect** in the top-right corner.
+3. Click **Get started** to proceed with wallet configuration.
+
+## Generating a wallet
+You'll be offered to import a pre-existing wallet. If you don't already have one, choose **Create a new wallet**.
+During the guided tutorial, you'll need to set a passphrase to protect your wallet. The passphrase
+is *not* the same as the recovery phrase. The passphrase is used to restrict access to the web wallet
+on your computer. The recovery phrase can be used to import your wallet on a fresh installation, or
+ona  different machine. Make sure to store both the passphrase and the recovery phrase
+securely, for example in a password manager.
+
+Re-enter portions of the recovery phrase when prompted, to confirm that you've saved it properly.
+Then you'll be taken to a screen that shows an initial synchronization process with the most
+recent testnet:
+
+<!--
+Do we want to maintain screenshots inside the web extension docs?
```

Comment:
Good question - presumably the UI will stabilize before long, at which point it will make more sense to maintain a screenshot, which also seems helpful enough to warrant including in the instructions. Let's ask the team about these candidate docs next week & ask about UI stability in this context

Reply:
In my view, it's OK for us to merge this one screenshot for now. I'll advocate for infrequent updates, and even eventually removing the screenshot maintenance from this repo. But let's get this in, so we can work an announcing the extension and getting more people using it.